ansible/roles-infra/infra-osp-resources-destroy/tasks/heat.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - environment:
     OS_AUTH_URL: "{{ osp_auth_url }}"
-    OS_USERNAME: "{{ osp_auth_username }}"
+    OS_USERNAME: "{{ guid }}-user"
```

Comment:
When destroying, don't we want to use the admin user ?

Reply:
This was a mistake on my part while experimenting that I didn't revert. Thanks for catching.